Hover/UI/HoverView.swift

```diff
@@ -248,13 +247,18 @@ private extension HoverView {
             }
         }
     }
-    
+
     private func animate(isOpening: Bool, anchor: Anchor, completion: (() -> Void)? = nil) {
         itemsStackView.isUserInteractionEnabled = isOpening
         
         UIViewPropertyAnimator(duration: Constant.animationDuration, curve: .easeInOut) {
             self.dimView.alpha = isOpening ? 1.0 : 0.0
         }.startAnimation()
+
+        let transform = imageExpandTransform(isOpening: isOpening)
+        UIViewPropertyAnimator(duration: Constant.animationDuration, dampingRatio: Constant.animationDamping) {
+            self.button.imageView.transform = transform
+        }.startAnimation()
```

Comment:
🤔 Have we considered reusing the existing `UIViewPropertyAnimator`?

```suggestion
        let transform = imageExpandTransform(isOpening: isOpening)
        UIViewPropertyAnimator(duration: Constant.animationDuration, curve: .easeInOut) {
            self.dimView.alpha = isOpening ? 1.0 : 0.0
            self.button.imageView.transform = transform
        }.startAnimation()
```

Reply:
I need the animation with the damping ratio, without it the animation looks weird.

Reply:
Unless you want to add the damping ratio for the `dimView` transition as well.

Reply:
> Unless you want to add the damping ratio for the dimView transition as well.

This should be fine, could you share a video with it to see how it looks? If it looks good, I don't see a problem with it 👍

Reply:
https://user-images.githubusercontent.com/987360/124335932-5ad19d80-db9c-11eb-9cf5-6c3fef56653e.mov

Reply:
Is it just me or does it look a bit weird? 🤔

Might be better to leave it as is for now - as you suggested

Reply:
It looks ok for me to be honest :)

Reply:
Let's go with it then, could be the video 👍

Reply:
The code is already up to date.